.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -5,17 +5,20 @@ Describe the pull request here, including any supplemental information needed to
 - expected change 1
 - expected change 2
 
+### This is how to verify this change...
+
 ### This pull request is ready to merge when...
 
-- [ ] Tests have been updated (and all tests are passing)
+- [ ] Develop has updated automated tests (and all tests are passing)
```

Comment:
is this supposed to be Developer?

Reply:
probably 😳

Reply:
I'll update that next week when we make the changes to the templates